src/helpers/delegation.ts

```diff
@@ -37,13 +37,19 @@ export async function getDelegators(network: string, address: string) {
   return await subgraphRequest(SNAPSHOT_SUBGRAPH_URL[network], params);
 }
 
-export async function getDelegatesBySpace(network: string, space: string) {
+export async function getDelegatesBySpace(
+  network: string,
+  space: string | undefined
+) {
+  const where = space
+    ? {
+        space
+      }
+    : {};
```

Comment:
@samuveth Passing `null` instead of empty string works with this new subgraph to get global delegations, sorry for not updating you about this :) didn't know we query it here too,

Reply:
Ok thanks